stores/json/store.go

```diff
@@ -42,15 +42,25 @@ func (store BinaryStore) EmitEncryptedFile(in sops.Tree) ([]byte, error) {
 	return store.store.EmitEncryptedFile(in)
 }
 
+const NOT_BINARY_HINT = ("This is likely not an encrypted binary file?" +
```

Comment:
https://google.github.io/styleguide/go/decisions.html#constant-names

Reply:
I guess I'm too used to const notation in C++, Java, Python, ... ;-)